pipeline/lib/util.ml

```diff
@@ -6,3 +6,14 @@ let get_branch_name branch =
   let prefix = "refs/heads/" in
   let len_prefix = String.length prefix in
   String.sub branch len_prefix (String.length branch - len_prefix)
+
+let current_timestamp () =
+  let two_weeks_in_unix_epoch = 1209600. in
+  let curr_gmt = Unix.gmtime (Unix.time () -. two_weeks_in_unix_epoch) in
+  let year = string_of_int curr_gmt.tm_year in
+  let month = string_of_int curr_gmt.tm_mon in
+  let day = string_of_int curr_gmt.tm_mday in
+  let hour = string_of_int curr_gmt.tm_hour in
+  let min = string_of_int curr_gmt.tm_min in
+  let sec = string_of_int curr_gmt.tm_sec in
+  year ^ "-" ^ month ^ "-" ^ day ^ "T" ^ hour ^ ":" ^ min ^ ":" ^ sec ^ "Z"
```

Comment:
I wish there was a better way to format timestamps in OCaml. :cry: 

There's a bunch of things that need to be taken care of here, looking at the docs for [`Unix.tm`](https://ocaml.org/api/Unix.html#TYPEtm), the code needs `year` to add `1900` and `month` to add `1`. 

Also, we need all the numbers to be zero padded. 

```ocaml
  let year = curr_gmt.tm_year + 1900 in
  let month = curr_gmt.tm_mon + 1 in
  let day = curr_gmt.tm_mday in
  let hour = curr_gmt.tm_hour in
  let min = curr_gmt.tm_min in
  let sec = curr_gmt.tm_sec in
  Format.sprintf "%04d-%02d-%02dT%02d:%02d:%02dZ" year month day hour min sec
```

Would something like this be better?